src/main/java/edu/cornell/kfs/module/cam/document/validation/impl/CuBarcodeInventoryErrorDocumentRule.java

```diff
@@ -0,0 +1,13 @@
+package edu.cornell.kfs.module.cam.document.validation.impl;
+
+import org.kuali.kfs.module.cam.document.BarcodeInventoryErrorDocument;
+import org.kuali.kfs.module.cam.document.validation.impl.BarcodeInventoryErrorDocumentRule;
+
+public class CuBarcodeInventoryErrorDocumentRule extends BarcodeInventoryErrorDocumentRule {
+
+    @Override
+    public boolean validateBarcodeInventoryErrorDetail(BarcodeInventoryErrorDocument document, boolean updateStatus) {
```

Comment:
Was it a functional requirement that this validation should be removed?

Reply:
No, however, to reach this point the Asset will not be found, if this wasn't overridden an error would occur and the document would route to error. See the base code version of the method in BarcodeInventoryErrorDocumentRule.java for more details.

Reply:
If we override this then the validation will not happen when the document is created manually either. Will the validation happen at a different point? I a not sure we just want to remove the validation. I will try to run this locally to better understand the issue you are running into. It may be good to have a zoom when you get in. Thank you!